webcompat/static/js/lib/models/issue.js

```diff
@@ -102,9 +91,14 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var regex = /\*\*Description\*\*: ([^*]+)/;
-    var description = regex.exec(body);
-    return description != null ? description[1].slice(0, 74) : null;
+    var el = document.createElement("html");
```

Comment:
@marimeireles can you explain what is happening here? It seems odd to create an `html` element.